docs/api/classes/CTERenamer.md

```
@@ -0,0 +1,221 @@
+
+The new name for the CTE.
+
+#### Returns
+
+`void`
+
+#### Throws
+
+When the specified CTE does not exist in the query.
+
+#### Throws
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)